CODE_OF_CONDUCT.md

```diff
@@ -73,8 +85,18 @@ affect a person's ability to participate within them.
   bad day, so please take this opportunity to question yourself, cool down if
   necessary and do not persist in the exact same behavior you have been
   reported for.
-
-
+- **Avoid using personal (member-to-member) communication.** 
```

Comment:
I'm concerned phrasing it like this will have a chilling effect on people who have concerns about behaviors.  What do you think of something like
```suggestion
- **Default to using public rather than personal (member-to-member) communication.**
```